doc/man/zcashd.1

```diff
@@ -79,7 +79,7 @@ mempool will accept (default: 0 = no limit applied)
 .HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-4\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-2\fR to 16, 0 = auto, <0 =
```

Comment:
Non-blocking: This has been commented on before in previous release processes. The number changing is intentional in `zcashd --help` since that is dynamic, but the manpage generated from it is necessarily static. Perhaps we should just adjust the release script to set this to some common number of cores? Or some large number.